src/pudl/transform/ferc1.py

```diff
@@ -115,6 +251,9 @@ class Config:
     )
     rename_columns_instant_xbrl: RenameColumns = RenameColumns()
     rename_columns_duration_xbrl: RenameColumns = RenameColumns()
+    wide_to_tidy_xbrl: WideToTidyXBRL = WideToTidyXBRL()
+    merge_metadata: MergeMetadata = MergeMetadata()
+    align_row_numbers_dbf: AlignRowNumbersDbf = AlignRowNumbersDbf()
```

Comment:
I think we should standardize with either ALL CAPS XBRL & DBF or CamelCase Xbrl & Dbf in all these class names. I think we've generally gone with CamelCase elsewhere.

Reply:
oh yeah i missed this on `WideToTidyXBRL`

Reply:
it feels a little unnatural to do this with acronyms, but i 100% agree that it should be standardize and the have defaulted to `WideToTidyXbrl` format